.travis.yml

```diff
@@ -7,6 +7,7 @@ notifications:
   - aqc2109@columbia.edu
   - mjr127@case.edu
 script:
+- npm run lintnofix
 - npm run build
```

Comment:
We shouldn't do `npm build:install` here, because `npm install` takes a lot more time while CI already does `npm ci` to install stuff.